src/Chainweb/Pact/PactService.hs

```diff
@@ -856,16 +856,26 @@ execValidateBlock
     => BlockHeader
     -> PayloadData
     -> PactServiceM cas PayloadWithOutputs
-execValidateBlock currHeader plData =
-    -- TODO: are we actually validating the output hash here?
-    withBatch $ withCheckpointerRewind mb "execValidateBlock" $ \pdbenv -> do
-        !result <- playOneBlock currHeader plData pdbenv
-        return $! Save currHeader result
+execValidateBlock currHeader plData = do
+    -- KILLSWITCH: The logic here involving `txSilenceDates` is to be removed in
+    -- a future version of Chainweb. This rejects any non-genesis block which is
+    -- found to contain transactions.
+    --
```

Comment:
I think, this is wrong. This logic has to stay. I must not be that a transaction that was invalid yesterday becomes valid tomorrow.

Otherwise, a selfish miner with a lot of hash power could mine a branch that has tx before the cut-off date and reveal the branch to the network after this code got remove.